### ISSUE TITLE
[ca] "soc"

### ISSUE THREAD

**Issue by tiff** [languagetool-org/languagetool#3671]
Feedback from a user:
> En catalán el monosílabo "soc" ja no se acentúa en ninguna circunstancia.

**Comment by jaumeortola** [comment]
Yes. There has been a modification in the spelling of a few words with diacritics. But we are still in the transition period, that ends next year. As this change has been a bit controversial, I don't see other solution that offering options to the users. But we don't have any way to do it easily. Any plans for this kind of options?

**Comment by jaumeortola** [comment]
I have reordered the rules in two rule groups: one with the old rules and one with the new rules. We could enable both groups by default, and the user could disable one of them (and keep the one they prefer).  

This would be a good solution for the browser add-on. The option "Turn off rule everywhere" exists in the browser add-on (and other UIs), but, unfortunately, it doesn't exist on the webpage or on the new editor.

**Comment by jaumeortola** [comment]
As an experiment, all diacritics rules have been enabled. This will create a loop (`soc->sóc->soc`). We suggest the users to disable the rules they don't want to use. This can be fine in the browser add-on because the choice is remembered with "Turn off rule everywhere". But in the webpage there is no such option.

**Comment by jaumeortola** [comment]
Now, my solution is enabled. It is an excellent solution for the browser add-on, but it is awful for the webpage. 

In the browser add-on, the user only needs two disable one rule once and for all. On the webpage, the user needs to disable the rules repeatedly for every different word.

**Comment by udomai** [comment]
This is not ideal... the "disable forever" option is not going to be added later. I think implementing an option in the settings for the old or new spelling rules will be the definitive solution — at some point. Until then, we might have to pick one spelling standard...

**Comment by jaumeortola** [comment]
> This is not ideal... the "disable forever" option is not going to be added later. 

In the browser add-on settings it can be re-enabled. I think this is good enough. 

> I think implementing an option in the settings for the old or new spelling rules will be the definitive solution — at some point. Until then, we might have to pick one spelling standard...

Agree. I will use it a couple of days to see how annoying it is now, and probably I will disable one of the rule groups.
